projects/packages/jetpack-mu-wpcom/src/features/import-customizations/import-customizations.php

```diff
@@ -0,0 +1,57 @@
+<?php // phpcs:ignore Squiz.Commenting.FileComment.Missing
+/**
+ * Customizations to the wp-admin/import.php page.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+/**
+ * Only add_action if the current screen is the wp-admin/import.php page.
+ */
+function import_page_customizations_init() {
+	$screen = get_current_screen();
+
+	if ( $screen && $screen->id === 'import' ) {
```

Comment:
Would it be better if we move this condition to `load_import_customizations`? 👀

Reply:
I'm unsure that would work because `load_import_customizations` is added during the `plugins_loaded` action. But our `get_current_screen` conditional happens on the `current_screen` action. So, if we moved it over to `load_import_customizations`, we'd still need to add a callback on the `current_screen` hook.

🤔 Maybe we could move the `wpcom_admin_interface` option conditional over to `load_import_customizations,` but I imagined maybe "import_customizations" could be expanded in the future outside of the scope of the admin interface style. Probably not, but that's where my thoughts were. 😆